src/breadcrumb-group/full-dropdown.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { Ref } from 'react';
+import clsx from 'clsx';
+
+import { CustomTriggerProps } from '../button-dropdown/interfaces';
+import InternalButtonDropdown from '../button-dropdown/internal';
+import InternalIcon from '../icon/internal';
+import { DATA_ATTR_FUNNEL_KEY, DATA_ATTR_RESOURCE_TYPE, FUNNEL_KEY_FUNNEL_NAME } from '../internal/analytics/selectors';
+import { CancelableEventHandler } from '../internal/events';
+import { BreadcrumbGroupProps } from './interfaces';
+
+import styles from './styles.css.js';
+
+interface FullCollapsedDropdownProps {
+  items: ReadonlyArray<BreadcrumbGroupProps.Item>;
+  onClick: CancelableEventHandler<{ id: string }>;
+  onFollow: CancelableEventHandler<{ id: string }>;
+}
+
+const metadataTypeAttribute = {
+  [DATA_ATTR_RESOURCE_TYPE]: 'true',
+};
+
+export const FullDropdown = ({ items, onClick, onFollow }: FullCollapsedDropdownProps) => (
+  <>
+    <InternalButtonDropdown
+      items={items.map((item, index) => {
+        const isCurrentPage = index === items.length - 1;
+        return {
+          id: index.toString(),
+          text: item.text,
+          href: isCurrentPage ? undefined : item.href,
+          disabled: isCurrentPage,
+          isCurrentPage,
+        };
+      })}
+      className={styles['collapsed-button-dropdown']}
```

Comment:
Should we add an internal prop to the button dropdown instead? The risk of having a custom style is that it is hard to secure no regressions when making changes to the button dropdown.